bigchaindb/consensus.py

```diff
@@ -89,6 +89,18 @@ def verify_signature(signed_transaction):
         """
         raise NotImplementedError
 
+    @abstractmethod
```

Comment:
I'm not sure I agree, but it should go in the refactor anyway.  It's probably out of the scope of this PR for me to change all the function signatures in `consensus.py`.